### ISSUE TITLE
Is the example of Nullable series in titanic.ipynb behave correct?

### ISSUE THREAD

**Issue by liyzcj** [dylan-profiler/visions#98]
**Describe the bug**

Currently, the example of Nullable series in titanic is:

```python
from visions.utils.series_utils import nullable_series_contains


class NullBiologicalSex(VisionsBaseType):
    @classmethod
    @nullable_series_contains
    def contains_op(cls, series):
        return series.hasnans and series in BiologicalSex
    
    @classmethod
    def get_relations(cls):
        return [IdentityRelation(cls, BiologicalSex)]
```

and the examples below return `False`:

```python
null_biosex_series = pd.Series([np.nan, 'male', 'female']) 
null_biosex_series in NullBiologicalSex

# Output False
```
Is that expected behavior? 

**To Reproduce**
1. Go to 'examples'
2. Run the `titanic.ipynb` example.

**Expected behavior**

In my opnion, the output shoud be `True` stand `null_biosex_series` is a `NullBiologicalSex` type.

**Additional context**

Maybe the  implementation of `contains_op()`  is not correct, the  `series.hasnans` should be removed.

```python
class NullBiologicalSex(VisionsBaseType):
    @classmethod
    @nullable_series_contains
    def contains_op(cls, series):
        return series in BiologicalSex
    
    @classmethod
    def get_relations(cls):
        return [IdentityRelation(cls, BiologicalSex)] 
```

**Comment by ieaves** [comment]
You are totally correct, thank you @liyzcj - #100 incorporates these changes with a small modification to your IdentityRelation.

```
class NullBiologicalSex(VisionsBaseType):
    @classmethod
    @nullable_series_contains
    def contains_op(cls, series):
        return series.isin({'male', 'female'}).all()
    
    @classmethod
    def get_relations(cls):
        return [IdentityRelation(cls, String)]

    
class BiologicalSex(VisionsBaseType):
    @classmethod
    def contains_op(cls, series):
        return series.isin({'male', 'female'}).all()
    
    @classmethod
    def get_relations(cls):
        return [IdentityRelation(cls, NullBiologicalSex, relationship=lambda s: not s.hasnans)]
```

**Comment by ieaves** [comment]
Resolved by #100
